terraform/groups/ecs-service/locals.tf

```diff
@@ -0,0 +1,78 @@
+# Define all hardcoded local variable and local variables looked up from data resources
+locals {
+  stack_name                  = "order-service" # this must match the stack name the service deploys into
+  name_prefix                 = "${local.stack_name}-${var.environment}"
+  global_prefix               = "global-${var.environment}"
+  service_name                = "certified-copies-orders-api"
```

Comment:
Just validating. I am not entirely sure.

Is the service name correct?
https://companieshouse.atlassian.net/wiki/spaces/Arch/pages/4217208873/ECS+Cluster+Architecture

This confluence page mentioned the service name as "certified-copies.orders.api.ch.gov.uk"

Just double check, if the microservice name is apt. If it is, please ignore the comment.

Reply:
I don't think that is the naming convention we followed. Tasks are named as:
`"${var.environment}-${var.service_name}"`, but no running task has a name of that format. 

`var.service_name` string is currently used both for service and task names, which  are both limited to 255 characters, and might get used elsewhere in the future. Keeping them as short as possible, while still distinctive, seems ideal.